src/toil/provisioners/abstractProvisioner.py

```diff
@@ -612,14 +629,17 @@ def addVolumesService(self, config: InstanceConfiguration):
             Type=oneshot
             Restart=no
             ExecStart=/usr/bin/bash /home/core/volumes.sh
+
+            [Install]
+            WantedBy=multi-user.target
```

Comment:
We need this because Ignition doesn't actually support the old `command` part of a unit, but just enables the unit, and we need it to start, right?

Reply:
Correct